src/Arcus.Templates.Tests.Integration/WebApi/WebApiProject.cs

```diff
@@ -196,15 +196,29 @@ public static WebApiProject CreateNew(TestConfig configuration, WebApiProjectOpt
             return project;
         }
 
+        /// <summary>
+        /// Verifies if the specified file is part of the project.
+        /// </summary>
+        /// <param name="filename">The complete path and filename of the file, relative to the root folder of the project (without leading (back)slash).</param>
+        /// <returns>True if the file is present, otherwise false.</returns>
+        public bool ContainsFile(string filename)
+        {
+            Guard.NotNullOrWhitespace(filename, nameof(filename), "The filename parameter cannot be null or empty");
```

Comment:
```suggestion
            Guard.NotNullOrWhitespace(filename, nameof(filename), "The filename parameter cannot be blank");
```

Hehe, it's whitespace, so also blank inputs.

Reply:
Does 'blank' also imply null ?

Reply:
😄 For me it does, it's more in the context of 'the model', as in "we are missing a value". How it is missing, (null or empty or something else...) doesn't add many info imho. But that's just me 😄 . It's a nit.